### PR TITLE
[v2] fixup! ASoC: rt1316/rt711-sdca: Add RT1316/RT711 SDCA vendor-specific driver

### PR DESCRIPTION
**Pull request by shumingfan**
fix some coding styles

### CODE REVIEW

**Review by RanderWang — APPROVED** [review]

**Review by bardliao — DISMISSED** [review]

File: sound/soc/codecs/rt711-sdca.h [hunk 1]
```diff
@@ -165,9 +165,9 @@ struct sdw_stream_data {
 #define RT711_BUF_ADDR_HID2			0x44030020
 
 /* RT711 SDCA function topology */
-#define FUN_JACK_CODEC 0x01
-#define FUN_MIC_ARRAY 0x02
-#define FUN_HID 0x03
+#define FUNCTION_JACK_CODEC 0x01
+#define FUNCTION_MIC_ARRAY 0x02
+#define FUNCTION_HID 0x03
```
Comment (plbossart) [inline sound/soc/codecs/rt711-sdca.h]: I think we are confusing things here.

This is not the topology, that's defined by the function type.
It looks like the Function number, which is device specific and used for addressing?

**Review by plbossart — CHANGES_REQUESTED** [review]
minor nit-pick below.

File: sound/soc/codecs/rt711-sdca.h (hunk 1 above)
Comment (shumingfan) [reply sound/soc/codecs/rt711-sdca.h]: > This is not the topology, that's defined by the function type.
> It looks like the Function number, which is device specific and used for addressing?

Yes. This function number was defined by Realtek and used for the hierarchical addressing.
This function number should not be the same as the function type.

Comment (shumingfan) [reply sound/soc/codecs/rt711-sdca.h]: I changed the definitions to 'FUNC_NUM_' for avoiding confusion.

**Review by plbossart — APPROVED** [review]

**Review by RanderWang — APPROVED** [review]

**Review by bardliao — APPROVED** [review]

**Pull request merged**